lxd/instance/drivers/driver_qemu.go

```diff
@@ -5598,6 +5598,47 @@ func (d *qemu) Update(args db.InstanceArgs, userRequested bool) error {
 		if newErr != nil {
 			return fmt.Errorf("Invalid root disk device: %w", newErr)
 		}
+
+		// If security.protection.start is being removed, we need to make sure that
+		// our root disk device is not attached to another instance.
+		if shared.IsTrue(oldExpandedConfig["security.protection.start"]) && shared.IsFalseOrEmpty(d.expandedConfig["security.protection.start"]) {
```

Comment:
Lets move this into a separate function to avoid increasing the Update functions' cyclomatic complexity any further.

Reply:
`changedConfig` would also pick up a `lxc config set v1 security.protection.start=true`; we only care about when the option is being removed/set `false`. This way we can trim a few queries off of setting the value to `true`.

Reply:
understood thanks